.github/workflows/build-website.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     # See https://github.com/actions/github-script
     - name: Extract imgur URL
       if: env.deploy_download_preview == 'true'
-      uses: actions/github-script@v5
+      uses: actions/github-script@9b7ebbb458768465bb03f28bbef5aa6b5d9d5baf # v5
```

Comment:
@jcfr It doesn't look like Dependabot has issued a PR for this yet. I was curious to see if it would issue an update that would change the comment to `# v6` or `# v6.4.0`. Both of those tags currently share the same commit SHA.

`actions/github-script@9b7ebbb458768465bb03f28bbef5aa6b5d9d5baf` corresponds to `# v5.1.1` https://github.com/actions/github-script/releases/tag/v5.1.1 and doesn't actually correspond to the current `# v5` tag which current corresponds to `# v5.2.0` https://github.com/actions/github-script/releases/tag/v5.

Reply:
@jcfr To followup a PR wasn't generated for this because https://github.com/Slicer/slicer.org/network/updates/629986042

![image](https://user-images.githubusercontent.com/15837524/226482260-33823626-2d44-4ed2-8a54-aadc7a299b07.png)


https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-dependabot-errors#dependabot-cannot-open-any-more-pull-requests